main.scss

```diff
@@ -35,12 +35,25 @@
 			color: oColorsGetColorFor('o-footer-dark-link', 'text');
 
 			&:hover,
-			&:focus {
+			&:focus, {
 				color: oColorsGetColorFor('o-footer-dark-link-hover', 'text');
 			}
 		}
 	}
 
+	.o-footer--theme-dark-hover {
```

Comment:
Are these styles just for the pa11y test? If so, should go into the demo CSS separately and not be named using the `o-footer` classes style.